frontend/libs/studio-components/src/components/StudioRedirectBox/StudioRedirectBox.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StudioRedirectBox, type StudioRedirectBoxProps } from './StudioRedirectBox';
+import { testRootClassNameAppending } from '../../test-utils/testRootClassNameAppending';
+
+const mockTitle: string = 'title';
+
+const defaultProps: StudioRedirectBoxProps = {
+  title: mockTitle,
+  children: <div />,
+};
+
+describe('StudioRedirectBox', () => {
```

Comment:
Ref. previous comment, then we can check that className is applied to root by using the test-utils for that. 😊